container-search-gui/src/main/resources/gui/icons/manifest.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Missing copyright.

Reply:
I thought JSON did not support comments (?)

Reply:
Ah, yes. Didn't notice. No license then.